executor.go

```diff
@@ -1706,9 +1765,16 @@ func (e *executor) mapperLocal(ctx context.Context, shards []uint64, mapFn mapFu
 	}
 }
 
+var translateCallCol = map[string]struct{}{
+	"Set":            {},
+	"Clear":          {},
+	"Row":            {},
+	"SetColumnAttrs": {},
+}
+
 func (e *executor) translateCall(index string, idx *Index, c *pql.Call) error {
 	var colKey, rowKey, fieldName string
-	if c.Name == "Set" || c.Name == "Clear" || c.Name == "Row" {
```

Comment:
I definitely like the idea of cleaning this up, but let's make it a switch statement instead of a map lookup:
```
switch c.Name {
case: "Set", "Clear", "Row", "SetColumnAttrs":
    ...
case: "SetRowAttrs":
    ...
default:
    ...
}
```

Reply:
@yuce I just merged #1679 which actually implements this `switch`, so you can just rebase master for the update, and add in `"SetColumnAttrs"`.

Reply:
Updated